src/apps/vpn/frontend/navigator.cpp

```diff
@@ -512,6 +515,19 @@ void Navigator::computeComponent() {
              topPriorityScreen->m_qmlComponent, ForceReloadAll);
 }
 
+void Navigator::requestScreenFromBottomBar(
+    Navigator::Screen requestedScreen, Navigator::LoadingFlags loadingFlags) {
+  // Exists so we can add glean metric for screen changes only from bottom bar
```

Comment:
I think this comment is unnecessary.

Reply:
I like to leave comments in any situation where a future maintainer many years down the line might say "not sure why they did this" and refactor some code into a worse state (in this case, I could see someone moving the metrics recording to `Navigator::requestScreen`, which would log events in many situations we don't want). My hope is to give clear reason as to why this exists the way it does.

Even with that philosophy (which is probably shared by many of us), I'll acknowledge this is a questionable case as to whether we need a comment. I usually err on the side of an extra comment, when unsure. 

If you think the comment should be removed, just say the word and I'll remove it.

Reply:
The code looks straight forward enough to not require the comment for the future, but no strong opinions.

Reply:
In that case I'm keeping it, as I don't think it harms.